include/elliptics/session.hpp

```diff
@@ -671,10 +679,18 @@ class session
 								uint32_t type, uint64_t flags,
 								const dnet_time& time_begin = dnet_time(),
 								const dnet_time& time_end = dnet_time());
+		async_iterator_result start_copy_iterator(const key &id, const std::vector<dnet_iterator_range>& ranges,
+								uint64_t flags,
+								const dnet_time& time_begin,
+								const dnet_time& time_end,
+								const std::vector<int> &dst_groups);
 		async_iterator_result pause_iterator(const key &id, uint64_t iterator_id);
 		async_iterator_result continue_iterator(const key &id, uint64_t iterator_id);
 		async_iterator_result cancel_iterator(const key &id, uint64_t iterator_id);
 
+		async_iterator_result server_send(const std::vector<std::string> &keys, uint64_t flags,
```

Comment:
Btw, you forgot to document other iterator functions, I will write doc comment for all